Add valid tokens for HTTP Server and Splunk HEC sources (#38358)
DataDog/documentation: content/en/observability_pipelines/sources/http_server.md

1. (Optional) Set up authentication tokens. See [Configure authentication tokens](#configure-authentication-tokens) for details.
1. Select the decoder you want to use on the HTTP messages. Your HTTP client logs must be in this format. **Note**: If you select `bytes` decoding, the raw log is stored in the `message` field.

### Optional settings

#### Enable TLS

{{% observability_pipelines/tls_settings %}}

{{% observability_pipelines/tls_settings_mtls %}}

#### Configure authentication tokens

If you store tokens as credentials in your HTTP request's authorization header, you can configure the Worker to check if incoming HTTP requests have a valid token. Request events that do not have a valid token are dropped. The Worker can also look up an endpoint path or an IP address instead of a header.

**Note**: You cannot configure authentication tokens with the {{< ui >}}Plain{{< /ui >}} authorization strategy.

To configure authentication tokens, enable the {{< ui >}}Configure authentication tokens{{< /ui >}} toggle:

1. Click {{< ui >}}Manage Tokens{{< /ui >}} and then {{< ui >}}Add Token{{< /ui >}}.

> 1. Enter the identifier for your token key.<br>**Note**: If you are using environment variables, the environment variable for this token is the identifier you entered.

Author: @20agbekodo could you confirm that this note is correct? Usually, we say if you enter an identifier and choose to use environment variables, the environment variable is the identifier entered, but prepended with `DD_OP_`. However, looking at your video, it doesn't seem like this is the case for authentication tokens.

Reviewer: It's also `DD_OP<identifier>`. On the video you can see that other env vars also don't include `DD_OP` in the input box for users to fill. The `DD_OP` would appear underneath in the Docker command / values.yaml

So it's safe to put `DD_OP` like other identifiers 👍

Author: Updating the note.

```suggestion
1. Enter the identifier for your token key.<br>**Note**: If you are using environment variables, the environment variable for this token is the identifier you entered prepended with `DD_OP_`.
```

> 1. (Optional) Enter a field and value if you want to add additional information to your logs because this token was used for validation.

Reviewer: ```suggestion
1. (Optional) Enter a field and value if you want to add additional information to logs successfully authenticated with this specific token.
```
You can change the phrasing ofc, the important part IMO is to convey more that this one specific token "matched"

(Same for Splunk HEC)
